src/sentry/flags/docs/api.md

```diff
@@ -83,7 +83,26 @@ Retrieve a single flag log instance.
   }
   ```
 
-## Signing Secret [/organizations/<organization_id_or_slug>/flags/hooks/provider/<provider>/signing-secret/]
+## Signing Secrets [/organizations/<organization_id_or_slug>/flags/hooks/signing-secrets/]
+
+### Browse Signing Secrets [GET]
+
+Browse a list of signing secrets. Secrets are unique per provider. Secrets only show the first six characters; the remainder are redacted.
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "createdAt": "2024-12-12T00:00:00Z",
+        "createdBy": 12345,
```

Comment:
is this the user ID?

Reply:
Sentry user id yes.  Different name preferred?

Reply:
@ryan953 is there a way we can get a user name from the id on the frontend?

Reply:
we could look up the user name on the server before returning the data, i think that's preferred to another round trip to the server.


but it looks like there is a `/users/$id` route as well that is implemented by `class UserDetailsEndpoint`